mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/mapping/SearchMappingBuilder.java

```diff
@@ -116,9 +116,14 @@ public CloseableSearchMapping build() {
 
 			SearchIntegrationFinalizer finalizer =
 					integrationPartialBuildState.finalizer( propertySource, propertyChecker );
+
+			SearchIntegrationEnvironment finalEnvironment = environment;
 			mapping = finalizer.finalizeMapping(
 					mappingKey,
-					(context, partialMapping) -> partialMapping.finalizeMapping()
+					(context, partialMapping) -> partialMapping.finalizeMapping(
+							finalEnvironment.propertySource(),
+							finalEnvironment.beanResolver()
+					)
```

Comment:
Please revert this, and get the configuration/bean resolver from the context in `StandalonePojoMapping#start`. See my other comment in `StandalonePojoMapping`.